docs/package.json

```diff
@@ -81,9 +81,9 @@
     "nprogress": "^0.2.0",
     "postcss": "^8.5.4",
     "prop-types": "^15.8.1",
-    "react": "^19.1.0",
+    "react": "19.0.0",
```

Comment:
Removed `^`, since otherwise subsequent install would again reinstall `19.1.0` if it exists in pnpm store.

Reply:
We can use `~19.0.0` in case a patch version is released

Reply:
Renovate will still suggest a patch release.
If we want to not see any info about updates, we need renovate rule.
Do you think we should add it?
I'd say that for now, let's wait and re-evaluate. 🤞